src/accordion/accordion.js

```diff
@@ -124,7 +124,7 @@ angular.module('ui.bootstrap.accordion', ['ui.bootstrap.collapse'])
     link: function(scope, element, attrs, controller) {
       scope.$watch(function() { return controller[attrs.uibAccordionTransclude]; }, function(heading) {
         if (heading) {
-          var elem = angular.element(element[0].querySelector('[uib-accordion-header]'));
+          var elem = angular.element(element[0].querySelector('[uib-accordion-header], [data-uib-accordion-header]'));
```

Comment:
Can you change it to support all of the selectors possible to be matched similar to how it works for the tabs? https://github.com/angular-ui/bootstrap/blob/master/src/tabs/tabs.js#L230-L236